src/sqlfluff/dialects/dialect_snowflake.py

```diff
@@ -3342,3 +3341,46 @@ class DescribeStatementSegment(BaseSegment):
             ),
         ),
     )
+
+
+@snowflake_dialect.segment(replace=True)
+class UseStatementSegment(BaseSegment):
+    """A `USE` statement.
+
+    USE ROLE <name>
+
+    USE WAREHOUSE <name>
+
+    USE [ DATABASE ] <name>
+
+    USE [ SCHEMA ] [<db_name>.]<name>
+
+    USE SECONDARY ROLES { ALL | NONE }
+
```

Comment:
Know you just copied from ANSI but do we need all these?

```suggestion
```

Reply:
not really, we already have the links to the docs so we can ditch these 👍